src/point_one/fusion_engine/messages/solution.h

```diff
@@ -204,8 +204,11 @@ struct alignas(4) PoseAuxMessage : public MessagePayload {
  */
 struct alignas(4) GNSSInfoMessage : public MessagePayload {
   static constexpr MessageType MESSAGE_TYPE = MessageType::GNSS_INFO;
-  static constexpr uint8_t MESSAGE_VERSION = 0;
+  static constexpr uint8_t MESSAGE_VERSION = 1;
```

Comment:
I would add a comment that last_differential_time was removed to add these new parameters and that this struct can be used to de-serialize a version 0 message as long as the new parameters are invalidated or not used.